app/src/constants/timeFormats.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Time formats.
+ *
+ */
+export enum TIME_FORMAT {
```

Comment:
Might be easiest to put this in the existing dateFormats.ts file.  Probably best to keep the date/time related stuff in one place.  The file name having 'date' in it isn't really an issue

Reply:
ok sounds good